packages/core/src/tools/error.ts

```diff
@@ -34,18 +36,25 @@ export enum ErrorHandling {
 // eslint-disable-next-line @typescript-eslint/no-redeclare
 export type ErrorSource = typeof ErrorSource[keyof typeof ErrorSource]
 
-export function formatUnknownError(stackTrace: StackTrace | undefined, errorObject: any, nonErrorPrefix: string) {
+export function formatUnknownError(
+  stackTrace: StackTrace | undefined,
+  errorObject: any,
+  nonErrorPrefix: string,
+  handlingStack?: StackTrace
+) {
   if (!stackTrace || (stackTrace.message === undefined && !(errorObject instanceof Error))) {
     return {
       message: `${nonErrorPrefix} ${jsonStringify(errorObject)!}`,
       stack: 'No stack, consider using an instance of Error',
+      handlingStack: handlingStack ? toStackTraceString(handlingStack) : undefined,
```

Comment:
From local test, it seems to be formatted as:
```
Error: 
  at func @ file:line:col
  ...
```
what about replacing `Error` by `Handled`?

Reply:
If we do that, should we also change "Error" by "Unhandled" for the other cases?

Reply:
For `Unhandled`, we don't have an `handling_stack`, right?
Not sure to get what you have in mind

Reply:
I don't really understand the goal of this tag to make a decision.

Reply:
I thought that this is what will be displayed in the UI but it may worth to sync with error tracking to see what they need in this field.

Reply:
Yes, I'll check with them

Reply:
Maxime said that `Error:` is okay for him.

Reply:
OK, did you check with someone from the error tracking app to ensure that the field structure is convenient for them?

Reply:
I've check with Eloy and I quote " if its a JS stack trace all that matters is that it has the same format and keywords as normal stack traces"